### ISSUE TITLE
Difference between the number of followers retrieved by stweet and the actual one.

### ISSUE THREAD

**Issue by antoinecomp** [markowanga/stweet#56]
I have noticed some difference in the number of followers provided by stweet compared to the actual ones on Twitter. Indeed, taking elonmusk as an example we should expect 48 million followers. However, we have 8171 with stweet:

```
import stweet as st

get_users_task = st.GetUsersTask(['elonmusk'])
users_collector = st.CollectorUserOutput()

st.GetUsersRunner(
    get_user_task=get_users_task,
    user_outputs=[users_collector]
).run()

users = users_collector.get_scrapped_users()

for user in users:
  print(user.followers_count)
```

It gives back `8171`

Last, thanks for creating this library! It's really great since twint is facing some issues atm.

**Comment by markowanga** [comment]
Hi, i checked this and I have bug in parsing, I will prepare the hotfix

**Comment by markowanga** [comment]
I have just released a new version of stweet `1.3.1` with this bug fix, thank you for report 😊
